suite-native/module-device-onboarding/src/hooks/useOnWalletRecapFinishedRedirect.tsx

```diff
@@ -0,0 +1,38 @@
+import { useCallback } from 'react';
+import { useSelector } from 'react-redux';
+
+import { useNavigation } from '@react-navigation/native';
+
+import { selectIsDeviceProtectedByPin } from '@suite-common/wallet-core';
+import {
+    DeviceOnboardingStackParamList,
+    DeviceOnboardingStackRoutes,
+    RootStackParamList,
+    StackToStackCompositeNavigationProps,
+} from '@suite-native/navigation';
+
+import { useOnDeviceOnboardingFinishedRedirect } from './useOnDeviceOnboardingFinishedRedirect';
+
+type NavigationProps = StackToStackCompositeNavigationProps<
+    DeviceOnboardingStackParamList,
+    DeviceOnboardingStackRoutes,
+    RootStackParamList
+>;
+
+export const useOnWalletRecapFinishedRedirect = () => {
```

Comment:
Since it uses navigation, would call it differently:
```suggestion
export const useOnWalletRecapFinishedNavigation = () => {
```

Reply:
Fixed